crypto/evp/p_lib.c

```diff
@@ -755,7 +758,7 @@ const unsigned char *EVP_PKEY_get0_hmac(const EVP_PKEY *pkey, size_t *len)
         ERR_raise(ERR_LIB_EVP, EVP_R_EXPECTING_AN_HMAC_KEY);
         return NULL;
     }
-    os = EVP_PKEY_get0(pkey);
```

Comment:
These should be implemented in similar way to EVP_PKEY_get0_RSA is. They shouldn't use the EVP_PKEY_get0().

Reply:
Should it be done in this PR or in a separate one?

Reply:
Done.